docker-base/Dockerfile

```diff
@@ -2,18 +2,19 @@
 # user. We split this from the main Dockerfile because this base doesn't change
 # and also because it kept breaking the build due to flakiness.
 FROM alpine:3.8
-LABEL authors="Anubhav Mishra, Luke Kysow"
+LABEL authors="Anubhav Mishra, Luke Kysow, Jocelyn Thode"
 
 # create atlantis user
-RUN addgroup atlantis && \
-    adduser -S -G atlantis atlantis
+RUN adduser -S -G root atlantis
 
-ENV ATLANTIS_HOME_DIR=/home/atlantis
+ENV HOME=/home/atlantis
 
 # install atlantis dependencies
 ENV DUMB_INIT_VERSION=1.2.0
 ENV GOSU_VERSION=1.10
-RUN apk add --no-cache ca-certificates gnupg curl git unzip bash openssh libcap openssl && \
+RUN chmod -R g=rwxs ${HOME} && \
+    chmod g=u /etc/passwd && \
```

Comment:
Can you move the `/etc/passwd` chmod up to the previous run instruction and remove the `HOME` chmod.

Reply:
Done.